apps/web/src/utils/bugsnag.ts

```diff
@@ -0,0 +1,57 @@
+// import React from 'react';
```

Comment:
🔪

Reply:
Yeah I know, usually we don't need it, and I'm not sure why, but I was getting a typescript error if I didn't import in this case. I'll futz around and see if i can remove in a follow-up